migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql

```diff
@@ -0,0 +1,411 @@
+-- hack_spuMAPFreeze
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRPROVIDER_FREEZE() 
```

Comment:
Add the Database to the procesure name
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRPROVIDER_FREEZE() 
```

Reply:
Done